setup.cfg

```diff
@@ -8,7 +8,7 @@ select = B,C,E,F,W,T4,B9
 
 [isort]
 known_first_party=xarray_schema
-known_third_party=dask,numpy,pkg_resources,pytest,setuptools,xarray
+known_third_party=numpy,pkg_resources,pytest,setuptools,xarray
```

Comment:
I refactored the chunks validation. In the end, I didn't  need the dask function anymore. Take a look at the implementation and let me know what you think.

Reply:
did we lose the checking for irregular chunks then? that function checked for irregular chunks before (comment is still there in L152 of `components.py`) which was necessary because we assume them to be regular in L159. does that now happen elsewhere?

Reply:
Fixed in a4173b2